cmake/machine-files/kokkos/hip.cmake

```diff
@@ -0,0 +1,5 @@
+# Settings used when HIP is the Kokkos backend
+
+#is this needed?
+set(Kokkos_ENABLE_AGGRESSIVE_VECTORIZATION FALSE CACHE BOOL "")
```

Comment:
Grepping the kokkos repo suggests this only matters for OpenMP and Threads backends.